### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by gtsiolis**
Include the following new sections:
- Documentation
- Questions
- Issues
- Related Projects
- Code of Conduct

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,3 +29,31 @@ Gitpod is provided as a [managed Saas version](https://gitpod.io) with a free su
...
+## Questions
+
+For questions and support please use the [community forum](http://community.gitpod.io/).
+Join the conversation, and connect with other community members. 💬
+
+You can also follow [`@gitpod`](https://twitter.com/gitpod) for announcements and updates from our team.
+
+## Issues
+
+The issue tracker is used for tracking **bug reports** and **feature requests** for the Gitpod open source project as well as planning current and future delopment efforts. 🗺️
```
Comment (gtsiolis) [inline README.md:47]: **nitpick**: Probably worth adding a link to the issue tracker.

```suggestion
The [issue tracker](https://github.com/gitpod-io/gitpod/issues) is used for tracking **bug reports** and **feature requests** for the Gitpod open source project as well as planning current and future delopment efforts. 🗺️
```

File: README.md [hunk 2]
```diff
@@ -29,3 +29,31 @@ Gitpod is provided as a [managed Saas version](https://gitpod.io) with a free su
...
+
+You can upvote [popular feature requests](https://github.com/gitpod-io/gitpod/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc) or [create a new one](https://github.com/gitpod-io/gitpod/issues/new?template=feature_request.md).
+
+## Related Projects
```
Comment (gtsiolis) [inline README.md:51]: **question**: Maybe add a reference for these projects once we move these to `gitpod-io` org? /cc @csweichel

File: README.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ Tightly integrated with GitLab, GitHub, and Bitbucket, Gitpod automatically and
 
 ## Features
 
- - Dev environment as code
+ - Dev environments as code
```
Comment (gtsiolis) [inline README.md:14]: **thought**: Maybe it's worth taking a fresh look at this list and update accordingly. Listing distinct features could also help.

**Review by gtsiolis — COMMENTED** [review]
@JohannesLandgraf WDYT?

File: README.md [hunk 4]
```diff
@@ -29,3 +29,31 @@ Gitpod is provided as a [managed Saas version](https://gitpod.io) with a free su
 Open any GitLab, GitHub, or Bitbucket project, by prefixing the project's URL with `gitpod.io/#`. 
 
 E.g. https://gitpod.io/#https://github.com/gitpod-io/gitpod
```
Comment (gtsiolis) [inline README.md:31]: **thought**: We could add a more expanded getting started section, following the update in the [getting started section in docs](https://www.gitpod.io/docs/getting-started/).

```suggestion
You can start using Gitpod with one or more of the following ways:

1. [Use a Prefixed URL](https://www.gitpod.io/docs/context-urls/)
1. [Install Browser Extension](https://www.gitpod.io/docs/browser-extension/)
1. [Enable GitLab Integration](https://www.gitpod.io/docs/gitlab-integration/)
1. Quick start using an [Example Project](https://www.gitpod.io/docs/getting-started/#example-project) or [OSS Project](https://www.gitpod.io/docs/getting-started/#gitpodified-open-source-project)
```

File: README.md (hunk 1 above)
Comment (ArthurSens) [inline README.md:47]:
```suggestion
The issue tracker is used for tracking **bug reports** and **feature requests** for the Gitpod open source project as well as planning current and future development efforts.
``` 
:stuck_out_tongue:

Comment (gtsiolis) [reply README.md:47]: Thanks for taking a look @ArthurSens! Do you also suggest removing the 🗺️ emoji from there?

Comment (ArthurSens) [reply README.md:47]: Whoopis, no! that was an accident :sweat_smile:

File: README.md [hunk 5]
```diff
@@ -29,3 +29,31 @@ Gitpod is provided as a [managed Saas version](https://gitpod.io) with a free su
...
+
+## Documentation
+
+All documentation can be found on https://docs.gitpod.io.
+For example, see [Introduction](https://www.gitpod.io/docs/) and [Getting Started](https://www.gitpod.io/docs/getting-started/) sections. 📚
+
+## Questions
+
+For questions and support please use the [community forum](http://community.gitpod.io/).
```
Comment (csweichel) [inline README.md:40]: Maybe we could specifically mention gitpod.io here, too. My hope would be that we can bring down the number of issues due to service issues.

File: README.md [hunk 6]
```diff
@@ -29,3 +29,31 @@ Gitpod is provided as a [managed Saas version](https://gitpod.io) with a free su
...
+## Issues
+
+The issue tracker is used for tracking **bug reports** and **feature requests** for the Gitpod open source project as well as planning current and future delopment efforts. 🗺️
+
+You can upvote [popular feature requests](https://github.com/gitpod-io/gitpod/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc) or [create a new one](https://github.com/gitpod-io/gitpod/issues/new?template=feature_request.md).
+
+## Related Projects
+
+During the development of Gitpod we had to also develop our own tooling infrastructure to make development easier and more efficient.
+To this extend we've developed a number of open source projects including [Werft](https://github.com/csweichel/werft), [Leeway](https://github.com/TypeFox/leeway), and [Dazzle](https://github.com/csweichel/dazzle/).
```
[the remark this answers is not in the document]
Comment (gtsiolis) [reply README.md]: Added in https://github.com/gitpod-io/gitpod/pull/2549/commits/2ddf3d17ed758586ebc8b52fdc207eb421c7e6ef using the updated links.

File: README.md (hunk 5 above)
Comment (gtsiolis) [reply README.md:40]: This was also the intent behind [this proposal](https://typefox.slack.com/archives/C01APSL1RPV/p1605108490052500) (internal).

**Comment by gtsiolis** [comment]
Let's merge this for now as this will improve `README` by helping community members finding appropriate resources.

Feature list improvements (see https://github.com/gitpod-io/gitpod/pull/2549#discussion_r543688670 cc @svenefftinge) and improvements related to service availability awareness (see https://github.com/gitpod-io/gitpod/pull/2549#discussion_r544465748 cc @csweichel) can be improved later.

**Comment by gtsiolis** [comment]
Could someone approve these changes or let me know if you think we need to add something before merging? /cc @JohannesLandgraf @csweichel @svenefftinge 

See [BEFORE](https://github.com/gitpod-io/gitpod/blob/master/README.md) and [AFTER](https://github.com/gitpod-io/gitpod/blob/gt/update-readme/README.md).

**Review by csweichel — APPROVED** [review]
Change LGTM.
Please squash the commits prior to merging.

**Comment by gtsiolis** [comment]
Thanks @csweichel! Checks are still failing but this shouldn't be related as this contains only `README` changes. Merging this.

**Comment by gtsiolis** [comment]
Let's try once more. 

/werft run

**Comment by csweichel** [comment]
> Thanks @csweichel! Checks are still failing but this shouldn't be related as this contains only `README` changes. Merging this.

I agree. The issue is disconcerting still. Chances are master will break/is broken.

**Comment by csweichel** [comment]
Nevermind the build failure ... it was actually [fixed on master](https://github.com/gitpod-io/gitpod/commit/a4a766791dd0077540884f0da3a1b21d158bf55a#diff-6f441456bcd7d8cd1a04ca3d3264a683c8d89e22fa600c6058ee0ac8c3c1ef80).

Please feel free to merge and sorry for the noise :)

**Comment by gtsiolis** [comment]
Thanks @csweichel! Rebased and force pushed again to try one last time. Will merge this afterwards regardless if checks pass.

**Pull request merged**